plugins/neo4j-graph-widget/components/WidgetNeo4jGraph.vue

```diff
@@ -120,17 +154,20 @@ export default {
         { value: GRAPHML, text: 'GraphML' },
```

Comment:
In props, for object, a good practice is to put a default value if its not required:
```
 props: {
    widget: {
      type: Object,
      default:()=>{}
    }
},
```
if required:
```
 props: {
    widget: {
      type: Object,
      required:true
    }
},
```